maestrowf/datastructures/core/study.py

```diff
@@ -164,7 +187,8 @@ class Study(DAG, PickleInterface):
     """
 
     def __init__(self, name, description,
-                 studyenv=None, parameters=None, steps=None, out_path="./"):
+                 studyenv=None, parameters=None, steps=None, out_path="./",
+                 draw=False):
```

Comment:
This might be a knit-pick, but it might be better to move `draw=False` to the `configure` method. That's where `throttle` and other settings are set -- to me it makes a bit more sense to have it there.

Reply:
There's also an associated place where we print settings:
```
[2020-08-18 10:29:02: INFO] [study: 460]
------------------------------------------
Submission attempts =       1
Submission restart limit =  1
Submission throttle limit = 0
Use temporary directory =   False
Hash workspaces =           False
Dry run enabled =           True
Output path =               /mnt/f/Code/Python/maestrowf/sample_output/lulesh/lulesh_sample1_20200818-102902
------------------------------------------
```

Reply:
Alright fixed the above comment:
```
------------------------------------------
Submission attempts =       1
Submission restart limit =  1
Submission throttle limit = 0
Use temporary directory =   False
Hash workspaces =           False
Dry run enabled =           True
DAG drawing enabled =       ['mpl-dot']
Output path =               /mnt/f/Code/Python/maestrowf/sample_output/lulesh/lulesh_sample1_20200818-121610
------------------------------------------
```